acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -121,3 +121,60 @@ function acquia_lift_profiles_update_7001() {
     menu_rebuild();
   }
 }
+
+/**
+ * Ensure all defined visitor actions are sync'd to Lift Web.
+ */
+function acquia_lift_profiles_update_7002(&$sandbox) {
```

Comment:
Neat - had to look this up to find out about multi-pass updates.  I had no idea that this was an option.  Definitely cool.